env/custom.sh

```diff
@@ -1,2 +1,3 @@
 #export CBIOPORTAL_URL="http://localhost:8081"
 #export GENOME_NEXUS_URL="https://www.genomenexus.org"
+BACKEND=cbioportal:demo-rfc72
```

Comment:
@inodb did this work?  i tried this but coun't get it to work (in another PR).  I mean, couldn't even get it to checkout the right branch

Reply:
this did not work, but updating `image:` directly does work